style.css

```diff
@@ -317,3 +317,87 @@ ul {
   height: 62px;
   border-bottom: 1px solid #dfe1e6;
 }
+
+.contact-section {
+  background: #6070ff url(./assets/images/contact-bg.svg) no-repeat 100% 100%;
+  height: max-content;
+  width: 375px;
+  border-top-left-radius: 80px;
+}
+
+.contact-h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  margin-top: 103px;
+}
+
+.contact-p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  margin-top: 12px;
+  width: 327px;
+  height: 102px;
+  margin-bottom: 87px;
+  text-align: center;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-form input {
+  height: 48px;
+  width: 327px;
+  border-radius: 8px;
+  margin-bottom: 20px;
+}
```

Comment:
- [ ] It is required to stick to the Figma design as close as possible. It seems you are missing the color for the text in the input fields. Kindly fix that. I believe the color value should be `#172b4d` else it will default to `#000` and we don't want that.
- [ ] Another thing we seem to have missed is that there's no spacing between the left border of the input fields and its text when filled. I suggest we fix that also.

<html>
<body>
<!--StartFragment-->

Figma design | Your design
-- | --
![image](https://user-images.githubusercontent.com/29071984/171295866-f2af394e-53ba-4f92-a651-35b4178d4bbd.png) |  ![image](https://user-images.githubusercontent.com/29071984/171295978-c4edf2a0-45b8-4cec-9ea0-fee0e864d5eb.png)


<!--EndFragment-->
</body>
</html>